docs/visual-basic/misc/bc32023.md

```diff
@@ -17,7 +17,8 @@ The group variable specified in a `For Each` statement is not a collection objec
   
 -   Declare the group variable to be of a class type that either supports the Visual Basic collection design or implements <xref:System.Collections.IEnumerable>.  
   
-## See Also  
- <xref:System.Collections.IEnumerable>  
- [For Each...Next Statement](../../visual-basic/language-reference/statements/for-each-next-statement.md)  
- [Visual Basic Collection Class](https://msdn.microsoft.com/library/0cb2d1ad-c58d-42c0-8e69-d81f5a15e532)
+## See also
+
+- <xref:System.Collections.IEnumerable>  
+- [For Each...Next Statement](../../visual-basic/language-reference/statements/for-each-next-statement.md)  
+- [Visual Basic Collection Class](../../visual-basic/programming-guide/concepts/collections.md#visual-basic-collection-class.md)
```

Comment:
The bookmark does not exist; the build system only generates bookmarks automatically for level-2 heads, not level-3 heads. Try the bookmark of the parent section: #kinds-of-collections

Reply:
I took the bookmark directly from the website page:
https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/collections#visual-basic-collection-class

But I see that I've added a wrong '.md' ext at the end of the bookmark